certify-default.properties

```diff
@@ -41,7 +41,7 @@ mosip.certify.supported.jwt-proof-alg={'RS256','PS256','ES256','ES256K','Ed25519
 #Crypto asymmetric algorithm name
 mosip.kernel.crypto.asymmetric-algorithm-name=RSA/ECB/OAEPWITHSHA-256ANDMGF1PADDING
 #Crypto symmetric algorithm name
-mosip.kernel.crypto.symmetric-algorithm-name=AES/GCM/PKCS5Padding
+mosip.kernel.crypto.symmetric-algorithm-name=AES/GCM/NoPadding
```

Comment:
This was done as per Taheer's recommendation.

ref: https://stackoverflow.com/questions/31248777/can-pkcs5padding-be-in-aes-gcm-mode